tools/cmake/app.cmake

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+# Enable fix for the Arm Compiler: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/6035
```

Comment:
Can the comment here instead (or also) describe that this policy is about?

Reply:
This is temporary and serves to force enable an unreleased feature. It'll be removed once CMake is updated. I'll make this PR ready for review then.

Reply:
It should be part of 3.21.0, hopefully it'll be out soon